packages/ui-a11y-utils/src/FocusRegion.ts

```diff
@@ -144,63 +125,59 @@ class FocusRegion {
     if (!this._active) {
       const doc = ownerDocument(this._contextElement)
 
-      // @ts-expect-error ts-migrate(2339) FIXME: Property '_keyboardFocusRegion' does not exist on ... Remove this comment to see the full error message
       this._keyboardFocusRegion.activate()
-      // @ts-expect-error ts-migrate(2339) FIXME: Property '_screenReaderFocusRegion' does not exist... Remove this comment to see the full error message
       this._screenReaderFocusRegion.activate()
 
-      // @ts-expect-error ts-migrate(2339) FIXME: Property '_options' does not exist on type 'FocusR... Remove this comment to see the full error message
       if (this._options.shouldCloseOnDocumentClick) {
         this._listeners.push(
-          // @ts-expect-error ts-migrate(2345) FIXME: Argument of type '{ remove(): void; }' is not assi... Remove this comment to see the full error message
-          addEventListener(doc, 'click', this.captureDocumentClick, true)
+          addEventListener(
+            doc,
+            'click',
+            this.captureDocumentClick as EventListener,
```

Comment:
I don't know why TS needs this cast, its issue is that `(e:MouseEvent)=>void` is not compatible with `(e:Event)=>void`